plugin/fzf.vim

```diff
@@ -262,7 +266,7 @@ try
     set shell=sh
   endif
 
-  if has('nvim') && len(filter(range(1, bufnr('$')), 'bufname(v:val) =~# ";#FZF"'))
+  if has('nvim') && len(filter(range(1, bufnr('$')), 'bufname(v:val) =~# ";#FZF"')) || exists('s:curr_job')
```

Comment:
Thank you for the review! Pushed the fix using `job_status(s:curr_job)`.

Unfortunately FZF still complains about already running instance after killing tmux pane. But this happens due to another bug in fzf-tmux, not because of this PR. I opened a separate issue #796 for this.

Once that bug is fixed, this branch will work correctly after killing tmux pane as well.